config/webhook.yaml

```diff
@@ -69,7 +69,7 @@ spec:
           value: tekton.dev/triggers
         ports:
         - name: metrics
-          containerPort: 9000
+          containerPort: 9090
```

Comment:
as controller is exposed on `9000` lets change this also to `9000` itself
/cc @dibyom @khrm